symplyphysics/laws/dynamics/instantaneous_power_is_force_times_velocity.py

```diff
@@ -0,0 +1,44 @@
+from sympy import Eq, cos
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+from symplyphysics.core.symbols.quantities import scale_factor
+
+# Description
+## Power is the rate at which the work is done by a force. The instantaneous power is the power
+## at a specific instant.
+
+# Law: P = F * v * cos(phi)
+## P - instantaneous power of force F
+## F - magnitude of force
+## v - speed
+## phi - angle between force and velocity vectors
+
+power = Symbol("power", units.power)
+force = Symbol("force", units.force)
+speed = Symbol("speed", units.speed)
+angle = Symbol("angle", angle_type)
+
+law = Eq(power, force * speed * cos(angle))
```

Comment:
Derive from https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/vector/instantaneous_power_is_force_dot_velocity.py

Reply:
I'm waiting for #324 to do this

Reply:
Add TODO and we will merge it.

Reply:
I finished the dot product one, by the way